hazelcast/pom.xml

```diff
@@ -738,6 +738,13 @@
             <version>${postgresql.version}</version>
             <scope>test</scope>
         </dependency>
+        <!-- Force higher version, compatible with M1 -->
+        <dependency>
```

Comment:
I guess it's a transitive dependency, could you add in the comment why we need it?

Reply:
Added explanation